### PR TITLE
Release 4.4.0

### PR DESCRIPTION
**Pull request by aleksandr-dorofeev**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are the files formatted correctly?
* [ ] ~Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?~

## Related PRs or issues

[AB#89325](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/89325)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,60 @@
...
+
+___
+
+## Version 4.4.0
 
 ### App Center
 
+#### iOS/Android
+
 * **[Breaking change]** Remove `AppCenter.SetCustomProperties` API.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:13]: These changes are related to UWP too

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,60 @@
...
 
+#### iOS
+
+* **[Fix]** Fix throw an exception when checking to authenticate MAC value during decryption.
+
 #### Android
 
+* **[Feature]** Add `AppCenter.setCountryCode(string)` API to set the country code manually.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,18 +1,60 @@
...
+* **[Fix]** Remove `android.support.test.InstrumentationRegistry` string that caused an error when checking applications on availability of android support libraries.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,18 +1,60 @@
...
 * **[Fix]** Fix crash in `AppCenterLoader` during call `getContext()` API when an application uses a custom context for initializing SDK. Such context should contain `getApplicationContext()` API which must return default Android application context.
+* **[Fix]** Fix print logs with `ASSERT` level.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,18 +1,60 @@
...
+* **[Fix]** Fix a crash during trying to get `startServiceLog` from the database after upgrading App Center SDK from the old versions.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,18 +1,60 @@
...
 
 ### App Center Analytics
 
+#### iOS/Android
+
  * **[Feature]** Add `Analytics.EnableManualSessionTracker` and `Analytics.StartSession` APIs for tracking session manually.
+ * **[Feature]** Increase the interval between sending logs from 3 to 6 seconds for the backend load optimization.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:28]: These changes are related to UWP too

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,18 +1,60 @@
...
+
+#### Android
+
+* **[Feature]** Add `AppCenter.setLogger` API to set custom logger.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,18 +1,60 @@
...
+
+#### iOS
+
+* **[Fix]** Cancel authorization process if application is not active, otherwise `ASWebAuthenticationSession` will fail opening browswer and update flow will end up being in a broken state. This only affects updating from a private distribution group.
+
+ ### App Center Crashes
+
+#### iOS
+
+* **[Feature]** Add `(NSString *)description` method to convert `MSACErrorReport` to a string and additional useful information about sending error.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,18 +1,60 @@
...
+* **[Fix]** Fix sending `Crashes.trackError` logs after allowing network requests after the launch app.
+* **[Improvement]** Update PLCrashReporter to 1.10.1.
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Not related to Unity

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,18 +1,60 @@
...
+* **[Feature]** Replace installing a new release using the deprecated intent action `ACTION_INSTALL_PACKAGE` with the `PackageInstaller` API.
+* **[Feature]** Remove the download manager task if the download doesn't start within 10 seconds.
+* **[Feature]** Add sumcheck on the downloaded file before starting the install process.
+* **[Fix]** Fix a crash after discarding the installation if the download of a new release was interrupted in the previous application start and resumed in the current one.
+
+#### iOS
+
+* **[Fix]** Cancel authorization process if application is not active, otherwise `ASWebAuthenticationSession` will fail opening browswer and update flow will end up being in a broken state. This only affects updating from a private distribution group.
+
+ ### App Center Crashes
```
Comment (AnastasiaKubova) [inline CHANGELOG.md]: Pls fix the formatting and move this part above Distribute

File: RELEASE.md [hunk 11]
```diff
@@ -1,3 +1,55 @@
+## Release 4.4.0
+
+### App Center
```
Comment (AnastasiaKubova) [inline RELEASE.md:3]: Copy changes from the changelog

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,18 +1,44 @@
...
+#### Android
+
+* **[Feature]** Replace installing a new release using the deprecated intent action `ACTION_INSTALL_PACKAGE` with the `PackageInstaller` API.
+* **[Feature]** Remove the download manager task if the download doesn't start within 10 seconds.
+* **[Feature]** Add sumcheck on the downloaded file before starting the install process.
+* **[Fix]** Fix a crash after discarding the installation if the download of a new release was interrupted in the previous application start and resumed in the current one.
+
+#### iOS
+
+* **[Fix]** Cancel authorization process if application is not active, otherwise `ASWebAuthenticationSession` will fail opening browswer and update flow will end up being in a broken state. This only affects updating from a private distribution group.
```
Comment (DmitriyKirakosyan) [inline CHANGELOG.md]: typo "browswer"

File: RELEASE.md [hunk 13]
```diff
@@ -1,3 +1,39 @@
...
+#### Android
+
+* **[Feature]** Replace installing a new release using the deprecated intent action `ACTION_INSTALL_PACKAGE` with the `PackageInstaller` API.
+* **[Feature]** Remove the download manager task if the download doesn't start within 10 seconds.
+* **[Feature]** Add sumcheck on the downloaded file before starting the install process.
+* **[Fix]** Fix a crash after discarding the installation if the download of a new release was interrupted in the previous application start and resumed in the current one.
+
+#### iOS
+
+* **[Fix]** Cancel authorization process if application is not active, otherwise `ASWebAuthenticationSession` will fail opening browswer and update flow will end up being in a broken state. This only affects updating from a private distribution group.
```
Comment (DmitriyKirakosyan) [inline RELEASE.md]: typo "browswer"

**Review by DmitriyKirakosyan — CHANGES_REQUESTED** [review]

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**